sdk/iot/Azure.Iot.Hub.Service/api/Azure.Iot.Hub.Service.netstandard2.0.cs

```diff
@@ -816,6 +851,45 @@ public partial class JobResponse
         public static bool operator !=(Azure.Iot.Hub.Service.Models.JobResponseType left, Azure.Iot.Hub.Service.Models.JobResponseType right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class MessageFeedbackBatch
+    {
+        internal MessageFeedbackBatch() { }
+        public System.DateTimeOffset? EnqueuedTime { get { throw null; } }
+        public string LockToken { get { throw null; } }
+        public System.Collections.Generic.IReadOnlyList<Azure.Iot.Hub.Service.Models.MessageResult> MessageResult { get { throw null; } }
+        public string UserId { get { throw null; } }
+    }
+    public partial class MessageResult
+    {
+        internal MessageResult() { }
+        public string DeviceGenerationId { get { throw null; } }
```

Comment:
what does this field indicate? (I checked ms docs - doesn't really explain well)